opentelemetry-appender-tracing/examples/basic.rs

```diff
@@ -24,5 +24,5 @@ fn main() {
     tracing_subscriber::registry().with(layer).init();
 
     error!(name: "my-event-name", target: "my-system", event_id = 20, user_name = "otel", user_email = "otel@opentelemetry.io");
-    drop(provider);
+    let _ = provider.shutdown();
```

Comment:
I think we should be promoting shutdown() instead of drop, as shutdown() has a feedback mechanism, so users know at least something went wrong with shutdown (and possibly do something else to indicate lack of clean shutdown)

@lalitb @TommyCpp any thoughts on this?

Reply:
Agree on explicit shutdown.

Reply:
Agree on this